v1/src/test/java/com/google/cloud/teleport/templates/TextToBigQueryStreamLT.java

```diff
@@ -0,0 +1,266 @@
+
+    PipelineLauncher.LaunchConfig options =
+        paramsAdder
+            .apply(
+                PipelineLauncher.LaunchConfig.builder(testName, SPEC_PATH)
+                    .addEnvironment("maxWorkers", 100)
+                    .addParameter("outputTable", toTableSpec(project, table))
+                    .addParameter("inputFilePattern", getTestMethodDirPath() + "/*")
+                    .addParameter("JSONPath", jsonPath)
+                    .addParameter(
+                        "bigQueryLoadingTemporaryDirectory", getTestMethodDirPath() + "/temp")
+                    .addParameter("javascriptTextTransformGcsPath", udfPath)
+                    .addParameter("javascriptTextTransformFunctionName", "identity"))
+            .build();
+
+    // Act
+    PipelineLauncher.LaunchInfo info = pipelineLauncher.launch(project, region, options);
+    assertThatPipeline(info).isRunning();
+    PipelineOperator.Result result =
+        pipelineOperator.waitForConditionAndFinish(
+            createConfig(info, Duration.ofMinutes(TIMEOUT_FOR_10_GB_TEST_MINUTES)),
+            BigQueryRowsCheck.builder(bigQueryResourceManager, table)
+                .setMinRows(NUM_MESSAGES_FOR_10GB)
+                .build());
+
+    // Assert
+    assertThatResult(result).meetsConditions();
+
+    // export results
+    exportMetricsToBigQuery(info, getMetrics(info, INPUT_PCOLLECTION, OUTPUT_PCOLLECTION));
+  }
+
+  private void testSteadyState1hr(
+      Function<PipelineLauncher.LaunchConfig.Builder, PipelineLauncher.LaunchConfig.Builder>
+          paramsAdder)
+      throws IOException, ParseException, InterruptedException {
+    // upload schema files and save path
+    String jsonPath =
+        getFullGcsPath(
+            ARTIFACT_BUCKET,
+            gcsClient
+                .uploadArtifact(
+                    "input/schema.json",
+                    Resources.getResource("TextIOToBigQueryTest/schema.json").getPath())
+                .name());
+
+    String udfPath =
+        getFullGcsPath(
+            ARTIFACT_BUCKET,
+            gcsClient
+                .uploadArtifact(
+                    "input/udf.js", Resources.getResource("TextIOToBigQueryTest/udf.js").getPath())
+                .name());
+
+    DataGenerator dataGenerator =
+        DataGenerator.builderWithSchemaTemplate(testName, "GAME_EVENT")
+            .setQPS("100000")
+            .setSinkType("GCS")
+            .setOutputDirectory(getTestMethodDirPath())
+            .setNumShards("20")
+            .setNumWorkers("10")
+            .setMaxNumWorkers("100")
+            .build();
+
+    /*
+     * This table will automatically expire 4000000 millis after creation if not cleaned up manually or by
+     * calling the {@link BigQueryResourceManager#cleanupAll()} method.
+     */
+    TableId table =
+        bigQueryResourceManager.createTable(testName, SCHEMA, System.currentTimeMillis() + 4000000);
```

Comment:
Change this to `System.currentTimeMillis() + 7200000` so it expires 2 hrs after creation? (also mention time in min/hours in comment instead of millis)

4000000 millis is 66 mins, which might be cutting it too close.